CMakeLists.txt

```diff
@@ -6,7 +6,8 @@ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/build)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/build)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/bin")
 project(${PROJECT_NAME})
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11")
+#set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11")
+set(CMAKE_C_STANDARD 11)
```

Comment:
`set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11")` would not change my version of C. It would remain to C99.

Reply:
@romancardenas I would appreciate if you could check if this change is okay.

Reply:
It seems to be a new functionality in CMake 3.1. I'm OK with the change